### DISCUSSION TITLE
Ability to request review from yourself on a PR

### DISCUSSION THREAD

**Discussion by Arinerron** [community/community#17048] [Pull Requests]
I want to be able to request a review from myself on a pull request I created so for tracking purposes.

It doesn't seem to be possible but would make my workflow less complicated since right now, I have to _remember_ which pull requests I've opened that I need to edit the writing in.

**Comment by gtirloni** [comment]
It makes no sense to request a review from yourself. What's the point of it?

Just add a `todo` label or something like that in the PR.

**Reply by Arinerron** [reply]
Would every member of my team then create a `todo-$name` label for themselves and have two separate methods of tracking who will review a PR? That seems silly.

A `todo` label only carries the piece of information that the PR needs to be reviewed, but not _who_ is assigned to review it. Being able to request a review from yourself communicates both of the following without requiring two separate methods of tracking this info:

* the PR review is TODO
* who is assigned to do it

**Reply by gtirloni** [reply]
Create `self-review` and filter your PRs.
